src/festim/hydrogen_transport_problem.py

```diff
@@ -384,19 +382,42 @@ def initialise_exports(self):
         a string, find species object in self.species"""
 
         for export in self.exports:
-            if isinstance(export, festim.VTXTemperatureExport):
-                self._temperature_as_function = (
-                    self._get_temperature_field_as_function()
-                )
-                self._vtxfiles.append(
-                    dolfinx.io.VTXWriter(
-                        self._temperature_as_function.function_space.mesh.comm,
-                        export.filename,
-                        self._temperature_as_function,
+            if isinstance(export, exports.ExportBaseClass):
+                if export.times:
+                    if not self.settings.stepsize.milestones:
+                        self.settings.stepsize.milestones = []
```

Comment:
Could this be done in `Stepsize` upon instanciation?

Reply:
Yep